rebasehelper/application.py

```diff
@@ -457,6 +458,12 @@ def _prepare_rebased_repository(self):
         if os.path.isfile(gitignore):
             shutil.copy(gitignore, self.rebased_sources_dir)
 
+        for source, _, source_type in self.spec_file.spc.sources:
+            if not urllib.parse.urlparse(source).scheme and source_type == 1:
+                source_path = os.path.join(self.execution_dir, source)
+                if os.path.isfile(source_path):
+                    shutil.copy(source_path, self.rebased_sources_dir)
+
```

Comment:
A nitpick, but I would move this to the top, before copying patches, just so it matches the usual ordering in a spec file.